ironfish/src/rpc/routes/wallet/getTransaction.ts

```diff
@@ -118,8 +122,10 @@ router.register<typeof GetAccountTransactionRequestSchema, GetAccountTransaction
 
     const notes = await getAccountDecryptedNotes(node, account, transaction)
 
+    const confirmations = request.data.confirmations ?? node.config.get('confirmations')
```

Comment:
getTransactonStatus already materializes this so I dont think we need this code in the RPC do we?

Reply:
I'm materializing it here only to include it in the response